test/go_vet/lint_package_format.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/bash
+
+# Number of files in this go package.
+fileCount=$(go list -f '{{len .GoFiles}}' $1)
+if [ $fileCount -eq 0 ]
```

Comment:
I think it's more standard to write `if [ $filecount -eq 0 ]; then` (on 1 line).